custom_components/ocpp/services.yaml

```
@@ -51,6 +51,23 @@ set_charge_rate:
       advanced: true
       example: '{"chargingProfileId":8,"stackLevel":0,"chargingProfileKind":"Relative","chargingProfilePurpose":"ChargePointMaxProfile","chargingSchedule":{"chargingRateUnit":"A","chargingSchedulePeriod":[{"startPeriod":0,"limit":16}]}}'
 
+trigger_custom_message:
+  name: Trigger Message  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)